wiki/Beatmap_ranking_procedure/de.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+  - qualification
+  - nomination
+  - nominations
+  - nom
+  - ranking
+  - ranked
+  - Qualifizierung
+  - Nominierung
+  - gerankt
+---
+
+# Ranking-Verfahren für Beatmaps
+
+*Siehe auch: [Rank (Begriffsabgrenzung)](/wiki/Disambiguation/Rank)*
+
+Bevor [Beatmaps](/wiki/Beatmap) in die Kategorie [Ranked](/wiki/Beatmap/Category#ranked) eingestuft werden können, müssen sie von der Community überprüft werden, indem sie das Ranking-Verfahren durchlaufen.
+
+## Feedback
+
+Der Ersteller einer Beatmap kann seine Einsendungen als `Work in Progress` oder `Ausstehend` kennzeichnen. Die osu!-Community soll Feedback zu Beatmaps geben, die den beiden Kategorien zugeordnet sind.
+
+Durch [Modding](/wiki/Modding) erhält man konstruktive Kritik für eine Beatmap und kann so die Qualität der Beatmap steigern. Üblicherweise veröffentlichen Nutzer ihr Feedback auf der [Diskussionsseite](/wiki/Beatmap_discussion) einer Beatmap oder reden mit dem Ersteller direkt über Probleme.
+
+Die Qualitätsstandards für die Ranked-Kategorie sind hoch, beim Modding müssen also viele Punkte beachtet werden. Manchmal kann es auch passieren, dass Beatmaps komplett neu gemacht werden müssen.
+
+Bevor die Beatmap für das Ranking nominiert wird, müssen mindestens 5 verschiedene Nutzer der Beatmap einen [Hype](/wiki/Beatmap/Hype) vergeben.
+
+## Nominierungen {id=nominations}
+
+::: Infobox
+![](img/nomination-progress-beatmapset1857663.png "Der Rankingstatus einer Beatmap auf der Diskussionsseite, wobei der Hype-Train und die Nominierungsleiste zum Teil gefüllt sind.")
+:::
+
+Eine **Nominierung** ist eine Bestätigung, dass eine Beatmap [gerankt](/wiki/Beatmap/Category#ranked) werden soll. Beatmaps, die eine angemessene Qualität aufweisen, erhalten nach einer Überprüfung eine solche Nominierung von Nutzern, die diese Nominierungen vergeben dürfen. Als Mindestanforderung gilt, dass Beatmaps den [Ranking-Kriterien](/wiki/Ranking_criteria) entsprechen müssen und 5 oder mehr [Hypes](/wiki/Beatmap/Hype) haben.
+
+Nominierungen werden oft von [Beatmap-Nominatoren](/wiki/People/Beatmap_Nominators) (*BN*), einer Gruppe erfahrener Modder, vergeben. Mitglieder des [Nomination Assessment Teams](/wiki/People/Nomination_Assessment_Team) (*NAT*) dürfen Beatmaps auch nominieren, jedoch ist das nicht ihre Hauptaufgabe.
```

Comment:
My German is failing me here, is it `ihre Hauptaufgabe` or `deren Hauptaufgabe`?

Reply:
Both are arguably fine, `ihre` seems more fitting here.